code/Documentation/MVP/conf.py

```diff
@@ -0,0 +1,188 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+#import os
+#import sys
+#sys.path.insert(0, os.path.abspath('.'))
+
+#from sphinx.builders.html import StandaloneHTMLBuilder
+import subprocess
+
+# Doxygen
+subprocess.call('doxygen Doxyfile.MVP', shell=True)
```

Comment:
Codacy found an issue: [subprocess call with shell=True seems safe, but may be changed in the future, consider rewriting without shell](https://app.codacy.com/gh/mommel/hs-lr-midi-schwurbler/pullRequest?prid=6419091)